rcudnn/cudnn-sys/Cargo.toml

```diff
@@ -19,4 +19,4 @@ libc = "0.2"
 
 [build-dependencies]
 pkg-config = "0.3"
-bindgen = "0.53"
+bindgen = "0.53.1"
```

Comment:
according to semver, this should work just fine without the `.1`, `cargo update` should update the `Cargo.lock` with the latest compat version to what is specified in `Cargo.toml`